### ISSUE TITLE
Support fusing BatchNorm after activation

### ISSUE THREAD

**Issue by silabs-Vladimir** [tensorflow/tflite-micro#1651]
Conv->BN->Activation is fused
Conv->Activation->BN is not fused

While the original BatchNorm-paper had BatchNorm before the activation, experiments have shown that BatchNorm after activation is actually better for model accuracy. 

See this for more info: https://github.com/ducha-aiki/caffenet-benchmark/blob/master/batchnorm.md
Also: https://www.reddit.com/r/MachineLearning/comments/67gonq/d_batch_normalization_before_or_after_relu/
